guide/getting-started/visual-language.md

```diff
@@ -171,7 +172,7 @@ Most likely stemming from the original depiction as a gold coin, there is a comm
 
 ### Illustrations
 
-Magazines, blogs, and other publications often hire professional illustrators to create custom visuals for specific stories. Additionally, companies that integrate or build on Bitcoin also frequently create unique marketing materials. Because Bitcoin does not have any official style, we see many different visual expressions in these illustrations.
+Magazines, blogs, and other publications often hire professional illustrators to visualize stores about bitcoin. Because Bitcoin does not have any official brand style guidelines, they are usually only limited by the illustrator and publisher’s imagination.
```

Comment:
```suggestion
Magazines, blogs, and other publications often hire professional illustrators to visualize stores about Bitcoin. Because Bitcoin does not have any official brand style guidelines, they are usually only limited by the illustrator and publisher’s imagination.
```